python/cuml/feature_extraction/text.py

```diff
@@ -189,10 +189,11 @@ def _compute_empty_doc_ids(self, count_df, n_doc):
         Compute empty docs ids using the remaining docs, given the total number
         of documents.
         """
-        remaining_docs = count_df['doc_id'].unique().values
-        doc_ids = cp.arange(0, n_doc)
-        empty_doc_ids = doc_ids[~cp.in1d(doc_ids, remaining_docs)]
-        return empty_doc_ids
+        remaining_docs = count_df['doc_id'].unique()
+        doc_ids = cudf.DataFrame(data={'all_ids': cp.arange(0, n_doc)})
```

Comment:
We should provide `dtype` here to be based `n_doc` to save on memory. 

We can probably use this  cudf util to figure it out based on `n_doc`. 
 https://github.com/rapidsai/cudf/blob/85e84d4118788d7b5e158b43ad4c7026f277d875/python/cudf/cudf/utils/dtypes.py#L218-L229

Reply:
Good point, I added your suggestion.

Reply:
Can we also change it here too ? 
```
all_ids: cp.arange(0, n_doc, dtype=cp.int32)
```

Reply:
On it.